src/filter.js

```diff
@@ -54,8 +54,9 @@ function extract(string, regex) {
 
 function matchFilter(tagList, toMatch) {
   return (toMatch.filter(function (tags) {
-    return (tags.filter(function (tag) {
+    const x =tags.filter(function (tag) {
       return tagList.includes(tag);
-    }).length > 0);
```

Comment:
This logic is wrong as for multiple tags, if even oine matches it will return as true and not work like a AND logic.